### DISCUSSION TITLE
Libfmt v8 required for conda installation

### DISCUSSION THREAD

**Discussion by chetanyagoyal** [The-OpenROAD-Project/OpenROAD#4162] [Q&A]
Trying to use 
openroad==2.0_10145_g0de6f5131 installed with conda gives the following error -
`openroad: error while loading shared libraries: libfmt.so.8: cannot open shared object file: No such file or directory`

This error occurs only on Ubuntu20.04 because libfmt version 8 does not exist for it. 
Are there any workarounds to this?

**Comment by rovinski** [comment]
I don't think we host a conda installation. The installer would be from a third party and you should ask them for support.

**Comment by maliberty** [comment]
@proppy is the conda installer something you support?

**Comment by proppy** [comment]
yes, it seems they already filed an issue over at https://github.com/hdl/conda-eda/issues/360 (so feel free to close the issue here).
